app/Http/Controllers/Product/ExtendedBaseProductController.php

```diff
@@ -133,12 +134,12 @@ public function getProductField(int $productid)
         }
     }
 
-    public function adminDownload($id, $invoice = '', $api = false)
+    public function adminDownload($id, $invoice = '')
     {
         try {
-            // $release = $this->getLinkToDownload($role, $invoice, $id);
-            $release = $this->downloadProductAdmin($id);
-            $name = Product::where('id', $id)->value('name');
+            if ($this->downloadValidation($allowDownLoad=true,$id,$invoice)) {
```

Comment:
php doesn't have a concept of named argument. You can replace $allowDownLoad=true by just true

Reply:
Fixed